cloud/scope/cluster.go

```diff
@@ -147,6 +140,21 @@ func (s *ClusterScope) LBSpecs() []azure.LBSpec {
 			Role:         infrav1.NodeOutboundRole,
 		},
 	}
+	if !s.IsIPv6Enabled() {
+		// for now no internal LB is created for ipv6 enabled clusters
```

Comment:
Does this mean that internal cluster traffic routes through the public LB?

Reply:
The internal LB is actually not being used for anything right now for ipv4 clusters (see https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/696). I plan to start using it as part of https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/486, but there is a bug for which a fix is currently being rolled out where check IP availability doesn't work when the vnet has an IPv6 address range, so I disabled the internal LB in the meantime for ipv6 until we can rationalize this stuff in #486 once the fix is rolled out completely (I already have a POC in my local branch).

Reply:
should drop a link to an issue tracking this in the comment?

Reply:
#952